tests/01http-server.pl

```diff
@@ -127,7 +127,9 @@ package SyTest::HTTPServer {
       my $path = uri_unescape $request->path;
 
       if( $CLIENT_LOG ) {
-         print STDERR "\e[1;32mReceived Request\e[m for $method $path:\n";
+         my $green = -t STDERR ? "\e[1;32m" : "";
```

Comment:
One of these days, I will stop assuming that the code you write had any intention behind it at all.

Done.